src/gui/mzroll/mzroll.pro

```diff
@@ -1,289 +1,292 @@
-                    spectramatching.h \
-                    mzfileio.h \
-                    logwidget.h \
-                    spectralhit.h \
-                    alignmentvizwidget.h \
-                    alignmentvizallgroupswidget.h \
-                   spectralhitstable.h\
-                    clusterdialog.h \
-                    loginform.h \
-                    aboutpolly.h \
-                    advancedsettings.h \
-                    peptidefragmentation.h \
-                    qcustomplot.h \
-                    saveJson.h \
-                    messageBoxResize.h \
-                    awsbucketcredentialsdialog.h \
-                    alignmentpolyvizdockwidget.h \
-                    controller.h \
-                    numeric_treewidgetitem.h \
-                    analytics.h \
-                    ElmavCrashHandler.h \
-                    isotopeplotdockwidget.h \
-                    gettingstarted.h \
-                    pollywaitdialog.h
-
-
-
-SOURCES += mainwindow.cpp  \
-database.cpp \
- plotdock.cpp \
- treedockwidget.cpp \
- node.cpp \
- enzyme_node.cpp \
- metabolite_node.cpp \
- tinyplot.cpp \
- graphwidget.cpp \
- pathwaywidget.cpp \
- settingsform.cpp \
- background_peaks_update.cpp \
- isotopeplot.cpp \
- barplot.cpp \
- boxplot.cpp \
- edge.cpp \
- point.cpp \
- history.cpp \
- spectrawidget.cpp \
- masscalcgui.cpp \
- adductwidget.cpp \
- isotopeswidget.cpp \
- ligandwidget.cpp \
- main.cpp \
- eicwidget.cpp \
- plot_axes.cpp \
- tabledockwidget.cpp \
- peakdetectiondialog.cpp \
- pollyelmaveninterface.cpp \
- comparesamplesdialog.cpp \
- traindialog.cpp \
- line.cpp  \
- heatmap.cpp \
- treemap.cpp \
- note.cpp  \
- suggest.cpp \
- animationcontrol.cpp\
- alignmentdialog.cpp\
- scatterplot.cpp \
- noteswidget.cpp \
- gallerywidget.cpp \
- QHistogramSlider.cpp \
- qdownloader.cpp \
- highlighter.cpp \
- rconsolewidget.cpp \
- clusterdialog.cpp \
- loginform.cpp \
- advancedsettings.cpp \
- aboutpolly.cpp \
- widgets/qprog.cpp \
- projectdockwidget.cpp \
- spectramatching.cpp \
- mzfileio.cpp \
- spectralhitstable.cpp \
- alignmentvizwidget.cpp \
- alignmentvizallgroupswidget.cpp \
- logwidget.cpp \
- peptidefragmentation.cpp \
- qcustomplot.cpp \
- saveJson.cpp \
-    awsbucketcredentialsdialog.cpp \
-    alignmentpolyvizdockwidget.cpp \
-    controller.cpp \
-    numeric_treewidgetitem.cpp \
-    analytics.cpp \
-    ElmavCrashHandler.cpp \
-    isotopeplotdockwidget.cpp \
-    gettingstarted.cpp \
-    pollywaitdialog.cpp
-
-
-contains (DEFINES,EMBEDHTTPSERVER) {
-    SOURCES += remotespectrahandler.cpp
-    HEADERS += remotespectrahandler.h
-    INCLUDEPATH += ../libpillow
-    LIBS += -lpillowcore
-}
-
-
-sources.files =  $$HEADERS \
-  $$RESOURCES \
-  $$SOURCES \
-  *.doc \
-  *.html \
-  *.pro \
-  images
- sources.path =  ./src
- target.path =  ../bin
-
-build_all {
-  !build_pass {
-   CONFIG -=    build_all
-  }
-}
+MOC_DIR=$$top_builddir/tmp/mzroll/

```

Comment:
@saifulbkhan why is the whole file showing up in diff?

Reply:
Strange. I only made additions to the SOURCES and HEADERS list. This should not be showing up.

Reply:
just make sure there are no other changes than what you intended. Cursory look didn't turn up anything problematic.

Reply:
I do not know why the entire file would turn up in diff, but I am pretty sure that no unintended changes are being included. Here's the diff for the file (before vs after) using an online tool: http://linediff.com/?id=5b991604687f4bfd7e8b4567